gen/dcos-config.yaml

```diff
@@ -4,6 +4,7 @@ package:
       {
         "environment": {
           "MESOS_IP_DISCOVERY_COMMAND": "/opt/mesosphere/bin/detect_ip",
+          "PLATFORM": "{{ platform }}"
```

Comment:
It doesn't need to be, @karlkfi you can remove this line. Adding it to the config is sufficient.